pkg/apis/triggers/v1alpha1/register.go

```diff
@@ -25,6 +25,7 @@ import (
 const (
 	GroupName              = "tekton.dev"
 	EventListenerLabelKey  = "/eventlistener"
+	EventIdLabelKey        = "/triggers-eventid"
```

Comment:
Nit: It should EventIDLabelKey

Check here:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
OK  I update.